csharp/upgrades/ad622770b3c38e7639883301e9e52ff1f3a4df4c/upgrade.properties

```diff
@@ -0,0 +1,2 @@
+description: Removed relations `is_constructed` and `is_generic`
+compatibility: full
```

Comment:
Would we also need
```
is_generic.rel: delete
is_constructed.rel: delete
```
Seems to not be needed but the documentation on https://wiki.semmle.com/display/IN/Creating+an+ODASA+Database+Upgrade+Script suggests that it should be there.